applications/GeoMechanicsApplication/custom_elements/drained_U_Pw_small_strain_element.hpp

```diff
@@ -51,7 +51,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) DrainedUPwSmallStrainElement
     ///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
     /// Default Constructor
```

Comment:
A very minor thing: just as you did for the other cases, please remove this comment.

Reply:
Done